.cirrus.yml

```diff
@@ -0,0 +1,18 @@
+freebsd_instance:
```

Comment:
please add a file comment providing some context & linking to the syntax for the file